conf/application.conf

```diff
@@ -85,58 +68,21 @@ controllers {
   # 300 is the default, you may need to change this according to your needs
   confidenceLevel = 300
 
-  com.kenshoo.play.metrics.MetricsController = {
-    needsAuth = false
-    needsLogging = false
-    needsAuditing = false
-  }
 }
 
 # Evolutions
 # ~~~~~
 # You can disable evolutions if needed
 # evolutionplugin=disabled
 
-# Metrics plugin settings - graphite reporting is configured on a per env basis
-metrics {
-  name = ${appName}
-  rateUnit = SECONDS
-  durationUnit = SECONDS
-  showSamples = true
-  jvm = true
-  enabled = true
-}
-
-# Microservice specific config
-
-auditing {
-  enabled=false
-  consumer {
-    baseUri {
-      host = localhost
-      port = 8100
-    }
-  }
-}
-
-
 mongo-async-driver {
   akka {
```

Comment:
does this akka need migrating to `org.apache.pekko`

Reply:
I'm guessing it's part of the underlying mongo async driver (not our hmrc one) so needs to be left alone.  Does make one think though.  But I could be wrong.